src/sql/workbench/services/connection/browser/connectionDialogService.ts

```diff
@@ -169,13 +170,15 @@ export class ConnectionDialogService implements IConnectionDialogService {
 				profile.serverName = trim(profile.serverName);
 
 				// append the port to the server name for SQL Server connections
-				if (this.getCurrentProviderName() === Constants.mssqlProviderName) {
+				if (this.getCurrentProviderName() === Constants.mssqlProviderName ||
```

Comment:
I'd suggest adding a comment here and elsewhere you do this letting people know why you're purposely using the MSSQL provider name if the provider name is MSSQL-CMS